pkg/cli/plugin_cmd.go

```diff
@@ -112,6 +113,19 @@ func getHelpArguments() []string {
 	return append(helpArgs, "-h")
 }
 
+// setupPluginEnv prepares some extra environment variables
+// that communicate certain information to plugins.
+func setupPluginEnv() {
+	env := make(map[string]string, 10)
```

Comment:
Why specifically the size of `10` here?

Reply:
Actually it technically should be `1` but when we add more variables, I was thinking we would not think to change that value, which would make things slightly less efficient, so I put `10` in preparation for 10 variables.

But as you know, none of this matters at the speeds we work with